### PR TITLE
Redirect Http-request to Https

### PR DESCRIPTION
**Pull request by hyungheo**
ALL of Http-request is automatically converted to  Https-request

### CODE REVIEW

**Comment by JayChl** [comment]
I got an error running server on ubuntu 64bit, 
below is the log any idea?

> node ./app.js

events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES 0.0.0.0:80
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at Server._listen2 (net.js:1249:19)
    at listen (net.js:1298:10)
    at Server.listen (net.js:1376:9)
    at Object.<anonymous> (/home/jclee/gits/absolute/server/app.js:47:13)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)

On Windows10(64 bit) is working fine

**Comment by hyungheo** [comment]
"80 Port" was needed root-permission to use at process in Ubuntu environment.
Use "sudo" command for run.cmd as below. 

sudo ./run.cmd

File: server/app.js [hunk 1]
```diff
@@ -34,14 +34,26 @@ app.use(function(err, req, res, next) {
 
 module.exports = app;
 
-var port = normalizePort(process.env.PORT || '3000');
-app.set('port', port);
+// PORT
+var http_port = normalizePort(process.env.PORT || '80');
```
Comment (romandev) [inline server/app.js]: I'm not sure if it is a good way to use 80(or 443) port. It's might already be being used in most of the systems.

**Comment by hyungheo** [comment]
It has been fixed. 
Thanks for your review

**Comment by hyungheo** [comment]
There's no problem when I run it on "OS X" and "windows10 x64"

**Pull request merged**